File: DotNetUEApp_C#/Models/Evaluation.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DotNetUEApp_C_.Models;

public class Evaluation
{
    public int Id { get; set; }
    public string Session { get; set; }
    public int Mois { get; set; }

    public string Type { get; set; }

    public string NoteFinal { get; set; }

    public int? PourcentageFinal { get; set; }

    public int? IdUniteEtude { get; set; }

    public string PourcentageRemediable { get; set; }


}

File: models/evaluation.ts
// evaluation.ts (dans le dossier models)

export class Evaluation {
    id!: number;
    session!: string;
    mois!: number;
    type!: string;
    noteFinal!: string;
    pourcentageFinal!: number | null;
    idUniteEtude!: number | null;
    pourcentageRemediable!: string;
  }
  